### PR TITLE
WIP Support retrieving status and configuration of feed triggers in CLI

### PR DESCRIPTION
**Pull request by abaruni**
This PR is to add support for retrieving the status and configuration of a feed trigger

Support has been implemented in the feed providers with the following:

https://github.com/apache/incubator-openwhisk-package-kafka/pull/217 <-- Merged
https://github.com/apache/incubator-openwhisk-package-cloudant/pull/137 <-- Merged
https://github.com/apache/incubator-openwhisk-package-alarms/pull/101 <-- Merged

### CODE REVIEW

**Comment by mdeuser** [comment]
@abaruni - when ready for review, please cc: @jessealva @drcariel @dubeejw and me.  thanks!

**Comment by abaruni** [comment]
🔵 PG3 1252

File: tools/cli/go-whisk-cli/commands/trigger.go [hunk 1]
```diff
@@ -326,18 +328,35 @@ var triggerGetCmd = &cobra.Command{
...
-        if (flags.trigger.summary) {
-            printSummary(retTrigger)
+        // Get full feed name from trigger get request as it is needed to get the feed
+        if retTrigger != nil && retTrigger.Annotations != nil {
+            fullFeedName = getValueString(retTrigger.Annotations, "feed")
+        }
+
+        if len(fullFeedName) > 0 {
+            fullTriggerName := fmt.Sprintf("/%s/%s", qualifiedName.GetNamespace(), qualifiedName.GetEntityName())
+            flags.common.param = append(flags.common.param, getFormattedJSON(FEED_LIFECYCLE_EVENT, FEED_READ))
```
Comment (dubeejw) [inline tools/cli/go-whisk-cli/commands/trigger.go]: Need a test for this.

**Comment by dubeejw** [comment]
@abaruni, comment 5 in this issue outlines the needed test for this PR. https://github.com/apache/incubator-openwhisk/issues/2921

**Comment by dubeejw** [comment]
Doc located here: https://github.com/apache/incubator-openwhisk/blob/master/docs/feeds.md#implementing-feed-actions.

**Comment by abaruni** [comment]
@jessealva can you take a look at this?

File: tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala [hunk 2]
```diff
@@ -1125,6 +1124,73 @@ class WskBasicUsageTests extends TestHelpers with WskTestHelpers {
     }
   }
 
+  it should "invoke a feed action with the correct lifecyle event when creating a feed trigger" in withAssetCleaner(
```
Comment (dubeejw) [inline tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala]: @rabbah, what are your thoughts on not using the `assetCleaner` to rolling these three tests into one using something like the below.
```
  it should "invoke a feed action with the correct lifecyle event when creating, reading, and deleting a feed trigger" in withAssetCleaner(
    wskprops) { (wp, assetHelper) =>
    val actionName = "echo"
    val triggerName = "feedTest"

    assetHelper.withCleaner(wsk.action, actionName) { (action, _) =>
      action.create(actionName, Some(TestUtils.getTestActionFilename("echo.js")))
    }

    try {
      wsk.trigger.create(triggerName, feed = Some(actionName)).
          stdout should include(""""lifecycleEvent": "CREATE"""")

      wsk.trigger.get(triggerName).
          stdout should include(""""lifecycleEvent": "READ"""")
    } finally {
      wsk.trigger.delete(triggerName).
          stdout should include(""""lifecycleEvent": "DELETE"""")
    }
  }
```

Comment (rabbah) [reply tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala]: the asset cleaner is a convenience - use it where it make sense.
your code looks good to me.

**Pull request merged**